core/templates/dev/head/domain/utilities/UrlInterpolationService.js

```diff
@@ -130,8 +130,8 @@ oppia.factory('UrlInterpolationService', [
           if (!value.match(VALID_URL_PARAMETER_VALUE_REGEX)) {
             AlertsService.fatalWarning(
               'Parameter values passed into interpolateUrl must only contain ' +
-              'alphanumerical characters, hyphens, underscores or spaces: \'' +
-              value + '\'');
+              'alphanumerical characters, hyphens, underscores, brackets or ' +
```

Comment:
Could we change "brackets" to "parentheses"? I think brackets typically mean "[" and "]".

Other than that, this looks great. Thanks very much @aks681 !

Reply:
Done!